src/main/kotlin/nl/joelchrist/shorty/shorties/domain/Shorty.kt

```diff
@@ -8,5 +8,6 @@ data class Shorty(@field:NotEmpty(message = "A shorty should have a full url")
                   @field:URL(message = "A shorty's full url should be a valid url")
                   val url: String = "",
                   val identifier: String? = null,
-                  val name: String = ShortyUtil().generateShortIdentifier(5),
-                  val owner: String? = null)
\ No newline at end of file
+                  val name: String = ShortyUtil.generateShortIdentifier(5),
+                  val owner: String? = null,
+                  val metadata: ShortyMetadata? = null)
```

Comment:
Ergens moet duidelijk worden dat het hier om een file gaat. Dit heet `metadata`, en in `metadata` vind ik alleen een `uuid` en een `url`. Mischien dat je daar juist `fileUrl` moet plaatsen, of dit object `fileMetadata` moet noemen.